Src/Fido2.Models/Objects/MakeNewCredentialResult.cs

```diff
@@ -5,16 +5,9 @@ namespace Fido2NetLib.Objects;
 /// <summary>
 /// Result of parsing and verifying attestation. Used to transport Public Key back to RP.
 /// </summary>
-public sealed class MakeNewCredentialResult : Fido2ResponseBase
+public sealed class MakeNewCredentialResult(RegisteredPublicKeyCredential credential)
 {
-    public MakeNewCredentialResult(string status, string errorMessage, RegisteredPublicKeyCredential? result)
-    {
-        Status = status;
-        ErrorMessage = errorMessage;
-        Result = result;
-    }
-
-    public RegisteredPublicKeyCredential? Result { get; }
+    public RegisteredPublicKeyCredential Credential { get; } = credential;
```

Comment:
I also took the opportunity to `.Result` to `.Credential`.

We could go even further and remove this class and just immediately return the credential -- that is more aligned to the Options-methods that immediately return Options.

My only concern is that this would generate even more of a breaking changes as well as make it hard to introduce/return any information that is not part of what we call the `RegisteredPublicKeyCredential`

Reply:
+ 1 for returning the credential directly and extending RegisteredPublicKeyCredential as needed.

Reply:
Alright, that was all the encouragement I needed. It's been done.